db/seeds.rb

```diff
@@ -1,7 +1,21 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+if Rails.env.development?
+
+  2.times do |i|
+    # Users
+    u = User.create!(
+                      username: Faker::Internet.user_name, 
+                      email: Faker::Internet.safe_email, 
+                      token: Faker::Internet.password(20)
+                    )
+    rand(10..20).times do |j|
+      # Gems
+      ruby_gem = RubyGem.create!(
+                                  name: Faker::App.name, 
+                                  info: Faker::Lorem.paragraph(2), 
+                                  project_uri: Faker::Internet.url('example.com')
+                                )
+      u.ruby_gems << ruby_gem
+      u.gems_adoptions.create!(ruby_gem_id: ruby_gem.id, description: Faker::Lorem.paragraph)
+    end
+  end
+end
```

Comment:
Hey @ramonh , it is failing because faker is generating the same `App.name` for more than  one `RubyGem`. Im going to fix it,  will let you know when I do.